README.md

```
@@ -27,6 +29,29 @@
 
 ---
 
+Every pod reported `2/2 Running`. `kubectl` said the cluster was healthy.
+47% of requests to one service were failing.
+
+Asked *"what is wrong with my system?"*, Rounds found it in 83 seconds — and
+showed its work:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)